backend/server/routes/realtime.ts

```diff
@@ -1,8 +1,11 @@
-const feathers = require('feathers');
+import * as feathers from 'feathers';
+import * as socketio from 'feathers-socketio';
+
+import UserService from '../services/user_service';
+
 const realtime = feathers();
-const socketio = require('feathers-socketio');
 
-realtime.configure(socketio({ wsEngine: 'uws' }, (io) => {
```

Comment:
Probably, I wasn't exactly sure what it did to begin with.  Someone recommended that as a more efficient socket engine. But it seems like we're not doing anything realtime soon.